packages/contracts-bedrock/scripts/deploy/Deploy.s.sol

```diff
@@ -145,7 +151,7 @@ contract Deploy is Deployer {
     }
 
     /// @notice Returns the proxy addresses, not reverting if any are unset.
-    function _proxiesUnstrict() internal view returns (Types.ContractSet memory proxies_) {
+    function _proxies() internal view returns (Types.ContractSet memory proxies_) {
```

Comment:
Do we no longer need the strict vs unstrict `_proxies` distinction?

Reply:
It would be very difficult to maintain, since we are deploying different contracts depending on the feature flags selected. My changes to ChainAssertions now have each function check for existence of the contract it is focused on, and I doubt that an undeployed contract would get past all of the other tests.

Reply:
Do you know the original motivation for it? I think not maintaining this is reasonable, as long as we understand what (if any) safety guarantees we're losing

Reply:
I recall its introduction in the test suite, I think for use in ChainAssertions primarily. It is currently not used anywhere outside of this test file.